<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraSelDlg</class>
 <widget class="QDialog" name="CameraSelDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(104, 98, 98,90);
border-radius:15px;
</string>
  </property>
  <widget class="QPushButton" name="pushButtonEnter">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>160</width>
     <height>218</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayoutVideo">
    <item row="5" column="0">
     <widget class="QCheckBox" name="checkBoxVideo2">
      <property name="text">
       <string>Video2</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBoxVideo6">
      <property name="text">
       <string>Video6</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="checkBoxVideo0">
      <property name="text">
       <string>Video0</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="checkBoxVideo1">
      <property name="text">
       <string>Video1</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBoxVideo7">
      <property name="text">
       <string>Video7</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBoxVideo4">
      <property name="text">
       <string>Video4</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBoxVideo5">
      <property name="text">
       <string>Video5</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="checkBoxVideo3">
      <property name="text">
       <string>Video3</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
